Makefile

```diff
@@ -6,6 +6,7 @@ NO_CACHE = false
 TCMALLOC = 
 PORTABLE = 0
 ARGS ?=
+TARGETPLATFORM ?= "linux/amd64"
```

Comment:
This is not needed as the argument TARGETPLATFORM is default and set based on the system arch. You do not need to specify it, as it will be set to the current value during the build. If the build uses amd64, it will be it, and the same for ARM. As you need to have `buildx` or arm docker to build this, id say not really something we want to add here.